store.go

```diff
@@ -1604,7 +1618,7 @@ func (s *store) ImageSize(id string) (int64, error) {
 		return -1, errors.Wrapf(err, "error reading list of big data items for image %q", id)
 	}
 	for _, name := range names {
-		n, err := imageStore.BigDataSize(id, name)
+		n, err := imageStore.BigDataSize(id, name, nil)
```

Comment:
This `nil` is not obviously safe AFAICS.  See the `imageStore.BigDataSize` comment first, though.